griptape/mixins/artifact_file_output_mixin.py

```diff
@@ -43,3 +43,16 @@ def _write_to_file(self, artifact: BaseArtifact) -> None:
             os.makedirs(os.path.dirname(outfile), exist_ok=True)
 
         Path(outfile).write_bytes(artifact.to_bytes())
+
+    def save_video_artifact(self, artifact: VideoArtifact) -> None:
+        if self.output_file:
+            outfile = self.output_file
+        elif self.output_dir:
+            outfile = os.path.join(self.output_dir, artifact.name + ".mp4")
+        else:
+            raise ValueError("No output_file or output_dir specified.")
```

Comment:
why this method? `_write_to_file` should be fine

Reply:
For some reason, it would call the to_text method inside of the base artifact class which the Video Artifact is unable to be converted to text in the same way.

Reply:
you need to override `to_bytes` on the vide artifact

Reply:
okay, that makes sense! Thank you!